.github/actions/pr-review/pr_review.py

```
@@ -970,6 +1019,30 @@ def judge_findings(
     return verified, True, excluded
 
 
+def coverage_gaps(units: list[DiffUnit], omitted: list[DiffUnit], parse_errors: list[str]) -> list[str]:
```

Ruff (0.16.1)

[warning] Unused function argument: `units`

(ARG001)